src/middleware/SecureTrafficTenantLevel.ts

```diff
@@ -0,0 +1,21 @@
+import * as builder from "botbuilder";
+const config = require("config");
+// Strip bot mentions from the message text
+export class SecureTrafficTenantLevel implements builder.IMiddlewareMap {
```

Comment:
Let's rename this to what it specifically does: RestrictIncomingMessagesToTenants
("Secure" is a bit broad.)

Reply:
Done.